cluster-provision/gocli/cmd/run.go

```diff
@@ -177,6 +186,31 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 		return err
 	}
 
+	prometheusEnabled, err := cmd.Flags().GetBool("enable-prometheus")
+	if err != nil {
+		return err
+	}
+
+	prometheusReplicas, err := cmd.Flags().GetUint("prometheus-replicas")
+	if err != nil {
+		return err
+	}
+
+	prometheusAlertmanagerEnabled, err := cmd.Flags().GetBool("enable-prometheus-alertmanager")
+	if err != nil {
+		return err
+	}
+
+	prometheusAlertmanagerReplicas, err := cmd.Flags().GetUint("prometheus-alertmanager-replicas")
+	if err != nil {
+		return err
+	}
+
+	grafanaEnabled, err := cmd.Flags().GetBool("enable-grafana")
```

Comment:
I didn't expect a Grafana instance inside the CI environment.
Not sure if I mind, but I thought the internal setup is just meant for exporting the metrics and the Grafana we'd use is an external one.
Having the option for testing manually is probably good though, I'd just not enable it for automated tests.

Reply:
Right, because of that grafana is disabled by default